Documentation/63501 add information about the new enterprise plans and how to try them in our enterprise guide (#18702)
opf/openproject: docs/enterprise-guide/README.md

## Enterprise add-ons

### Preview of Enterprise plan changes (effective 2025-05-21)

> With the release OpenProject 16.0.0 on May 21st, 2025, we will update our Enterprise plans and with it the containing add-ons.

Reviewer: ```suggestion
With the release of OpenProject 16.0.0 on May 21st, 2025, we will update our Enterprise plans and with it the containing add-ons.
```

> Customers who have booked an Enterprise plan before or who will book a plan before the upcoming changes will not be effected by any changes and will keep the existing plans also after renewal.

Reviewer: ```suggestion
Customers who have booked an Enterprise plan before May 21, 2025, will not be effected by any changes and will keep the existing plans also after renewal.
```

Reviewer: Changed it to be less complicated and more specific (with the date)

> However, new add-ons that will be released with 16.0.0 and after in the higher Enterprise plans will not be available in these old plans. You can test these new add-ons in a trial.

Author: ```suggestion
However, new add-ons that will be released with 16.0.0 and afterwards in higher Enterprise plans will not be available in these lower old plans. You can test these new add-ons of higher plans in a trial.
```

> However, new add-ons that will be released with 16.0.0 and after in the higher Enterprise plans will not be available in theses old plans. You may test these new add-ons in a trial.

Reviewer: ```suggestion
However, new add-ons that will be released with 16.0.0 and after in the higher Enterprise plans will not be available in these old plans. You can test these new add-ons in a trial.
```

Reviewer: One more thought: Do we need the part "in the higher Enterprise plans"? Will add-ons that are part of the Basic plan be available for existing customers? If no, we should delete this part. If yes, we might clarify this more.

Reviewer: If yes, suggestion:

However, new add-ons that will be released with 16.0.0 and after in the higher Enterprise plans (Professional, Premium, Corporate) will not be available in these old plans. You can test these new add-ons in a trial. Also, customers in old plans will benefit from upcoming add-ons that are part of the Enterprise Basic plan.

> The following changes to our Enterprise plans will become effective on 2025-05-21 and apply for new customers:

Reviewer: ```suggestion
The following changes to our Enterprise plans will become effective on 2025-05-21 and **apply for new customers**:
```

Reviewer: to strengthen that again

| Community                                                    | Basic                                                        | Professional                                                 | Premium                                                      | Corporate                                                    |
| ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |

> | **All [Community features](https://www.openproject-edge.com/pricing/#features)** | **All Community features +**<br />**Basic Enterprise add-ons**:<br />- Baseline comparison<br />- Team boards<br/>- Version boards<br/>- Subproject boards<br/>- Work Breakdown Structure<br/>- Kanban boards (Status)<br/>- Attribute highlighting<br/>- Intelligent workflows with custom action buttons<br/>- Custom fields of type hierarchy<br/>- Date alerts<br/>- Custom theme and logo<br/>- Configure work package forms<br/>- Gantt charts PDF export<br/>- Graphs on project overview page<br/>- Placeholder users<br/>- Read-only mode for work packages<br/>- Team planner<br/>- Display relations in the work package table | **All Basic +**<br />**Professional Enterprise add-ons**:<br />- Single sign-on<br />-  OneDrive/SharePoint integration<br/>- Share work packages with external users<br/>- Automatic work package subject generation (coming with 16.0.0)<br />- Internal work package comments (coming with 16.0.0) | **All Professional +**<br />**Premium Enterprise add-ons**:<br />- Share project lists (project portfolio)<br />- LDAP users and group sync<br />- Customize project life cycle (coming with 16.0.0) | **All Premium +**<br />**Corporate Enterprise add-ons**:<br/>- OIDC & JWTs (coming with 16.0.0) <br />- Nextcloud Hub integration<br/>- openDesk integration<br/>- Antivirus scanning |

Reviewer: What are Team boards? Assignee boards? If yes, I would at least add this term in brackets like with the Status board. And Parent-child is missing? Also, I would list all boards together. Currently, we have:
- Baseline comparison
- Team boards
- Version boards
- Subproject boards
- Work Breakdown Structure
- Kanban boards (Status)

Author: - Kanband boards (Status)
- Team boards (Assignee)
- Product boards (Version)
- Project boards (Subproject)
- Work Breakdown Structure (Parent-child)

--> Überschrifte noch in den Docs anpassen.

### Testing the new Enterprise add-ons

**Enterprise on-premises**

> If you are currently on a lower plan for the Enterprise on-premises and want to test any of the new released Enterprise add-ons from the higher plans, please [get in touch with our Sales team](https://www.openproject.org/contact/) to get a new Enterprise trial token.

Author: ```suggestion
If you are currently on a lower plan for the Enterprise on-premises (e.g. Basic plan) and want to test any of the new released Enterprise add-ons from the higher plans (e.g. Professional plan), please [get in touch with our Sales team](https://www.openproject.org/contact/) to get a new Enterprise trial token.
```

> If you are currently on a lower plan for the Enterprise on-premises and want to test any of the new released Enterprise add-ons from the higher plans, please [get in touch with our Sales team](https://www.openproject.org/contact/) to get a new Enterprise trial token.

Reviewer: I know this will be different after 16.0, but right now, the term "in a lower plan" might confuse existing customers. Maybe change to:

If you are currently on a lower or old plan for Enterprise on-premises and want to test any of the new released Enterprise add-ons from the higher plans (Professional, Premium, Corporate), please [get in touch with our Sales team](https://www.openproject.org/contact/) to get a new Enterprise trial token.
